examples/deno/CommunityToolkit.Aspire.Hosting.Deno.Apphost/CommunityToolkit.Aspire.Hosting.Deno.Apphost.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsAspireHost>true</IsAspireHost>
+    <UserSecretsId>68a53d20-c63f-4188-8bf5-5206263fa5c7</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Aspire.Hosting.AppHost" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\src\CommunityToolkit.Aspire.Hosting.Deno\CommunityToolkit.Aspire.Hosting.Deno.csproj" IsAspireProjectResource="false" />
+  </ItemGroup>
+
+  <Target Name="CleanNodeModules" AfterTargets="Build">
+    <!--<Exec WorkingDirectory="../vite-demo" Command="rm -r node_modules" Condition="Exists('../vite-demo/node_modules')" />-->
+  </Target>
```

Comment:
If this isn't needed, let's remove it.

```suggestion
```

Reply:
This should now be using Delete task and does not cause build errors for me anymore.